src/parser/regex/execRegex.spec.ts

```diff
@@ -21,4 +21,9 @@ describe('execRegex Tests', () => {
     const result = execRegex(value, { pattern: '\\d+' });
     expect('26').to.deep.equal(result);
   });
+  it('Case 4: Define it as a regex object', () => {
+    const value = 'omer 1907';
```

Comment:
Open source project contributions should not contain personal information. For example, more general and common examples can be used here.

`90m2`
`George Orwell - 1984`